### PR TITLE
[#60] Fix: The Coin prices period text color show incorrect on Light mode

### PR DESCRIPTION
**Pull request by hoangnguyen92dn**
https://github.com/nimblehq/jetpack-compose-crypto/issues/60

## What happened 👀

Update the text color in Light Mode for the Coin price interval button

## Insight 📝

Add the text color to `Color.kt` to supported Light/Dark mode

## Proof Of Work 📹

|Dark|Light|
|-|-|
|image|image|

### CODE REVIEW

File: app/src/main/java/co/nimblehq/compose/crypto/ui/components/chartintervals/ChartIntervalsButton.kt [hunk 1]
```diff
@@ -60,6 +67,7 @@ fun ChartIntervalsButtonGroup(
 @Composable
 fun ChartIntervalsButton(
     modifier: Modifier,
+    textColor: androidx.compose.ui.graphics.Color,
```
Comment (luongvo) [inline app/src/main/java/co/nimblehq/compose/crypto/ui/components/chartintervals/ChartIntervalsButton.kt]: please optimize the import 🙏

File: app/src/main/java/co/nimblehq/compose/crypto/ui/components/chartintervals/ChartIntervalsButton.kt [hunk 2]
```diff
@@ -38,13 +38,20 @@ fun ChartIntervalsButtonGroup(
...
+            }
+
             ChartIntervalsButton(
                 modifier = Modifier
                     .requiredWidth(Dp45)
                     .background(
                         color = backgroundColor,
                         shape = RoundedCornerShape(Dp12)
                     ),
+                textColor = textColor,
```
Comment (luongvo) [inline app/src/main/java/co/nimblehq/compose/crypto/ui/components/chartintervals/ChartIntervalsButton.kt]: Should we make use of `coinPriceIntervalTextColor`?

Comment (hoangnguyen92dn) [reply app/src/main/java/co/nimblehq/compose/crypto/ui/components/chartintervals/ChartIntervalsButton.kt]: @luongvo Ops. `coinPriceIntervalTextColor` is redundant. Let me remove it.

**Review by luongvo — APPROVED** [review]
💯

**Review by Wadeewee — APPROVED** [review]

**Pull request merged**